libs/movie/src/lib/movie/+state/movie.service.ts

```diff
@@ -14,18 +13,18 @@ import { App } from '@blockframes/utils/apps';
 import { QueryFn } from '@angular/fire/firestore';
 import { OrganizationService } from '@blockframes/organization/+state';
 import { map } from 'rxjs/operators';
-import { getViews } from '../pipes/analytics.pipe';
 import { joinWith } from '@blockframes/utils/operators';
-import { AnalyticsService } from '@blockframes/utils/analytics/analytics.service';
+import { AnalyticsService } from '@blockframes/analytics/+state/analytics.service';
 import { AuthService } from '@blockframes/auth/+state';
 import { ActiveState, EntityState } from '@datorama/akita';
+import { Analytics } from '@blockframes/analytics/+state/analytics.firestore';
 
 export const fromOrg = (orgId: string): QueryFn => ref => ref.where('orgIds', 'array-contains', orgId);
 export const fromOrgAndAccepted = (orgId: string, appli: App): QueryFn => ref => ref.where(`app.${appli}.status`, '==', 'accepted').where('orgIds', 'array-contains', orgId);
 export const fromOrgAndInternalRef = (orgId: string, internalRef: string): QueryFn => ref => ref.where('orgIds', 'array-contains', orgId).where('internalRef', '==', internalRef);
 export const fromInternalRef = (internalRef: string): QueryFn => ref => ref.where('internalRef', '==', internalRef);
 
-type MovieWithAnalytics = Movie & { analytics: MovieAnalytics };
+type MovieWithAnalytics = Movie & { analytics: Analytics<'title'>[] };
```

Comment:
Use interface when extending another interface.
```suggestion
interface MovieWithAnalytics extends Movie {
  analytics: Analytics<'title'>[];
}
```